### DISCUSSION TITLE
Way to enter supercharger cost manually for an accurate trip cost

### DISCUSSION THREAD

**Discussion by seatadpole** [teslamate-org/teslamate#2660] [Q&A] [answered]
Is there a table where I could enter the exact DC charging cost to get an accurate trip cost? My understanding is that there is no automatic way for Teslamate to pull the data from Tesla. I do have TeslamateAgile installed with $0.13/kWh fixed rate and that's how I am getting a value (for AC charging only) here. Thanks!

![](https://i.imgur.com/rLzJv4D.png)

**Comment by DrMichael** [comment, marked as answer]
You can set the total charging costs in the Charges dashboard. ($0.13/kWh?!?! I want that too! :-)

**Comment by seatadpole** [comment]
That was too easy! Appreciate the quick tip 🙏
It's actually closer to 12¢ in VA but I just rounded it up to be on the safer side 🙂
